### PR TITLE
limit_field tests

### PR DESCRIPTION
**Pull request by Sild**
tests for limit field (2946)

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/property/PropertyMethod.java [hunk 1]
```diff
@@ -156,4 +157,9 @@ protected Response deleteProperty(final Map deleteObj) throws IOException {
         }};
         return httpApiResource.path(METHOD_PROPERTY_DELETE).request().post(Entity.entity(jsonArray.toJSONString(), MediaType.APPLICATION_JSON_TYPE));
     }
+
+    protected int calculateJsonArraySize(String jsonArray) throws IOException {
```
Comment (rmakulov) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyMethod.java]: What about  use instead?
        JSONArray array = new JSONArray(jsonArray);
        return array.length();

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyMethod.java]: There is no constructor to build it from String:

image

**Comment by KutuzovGregory** [comment]
ok

File: src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java [hunk 2]
```diff
@@ -1285,4 +1285,134 @@ public void testKeyTagExpressionUpperKey() throws Exception {
...
+
+        final Property property3 = new Property();
+        property3.setType(property.getType());
+        property3.setEntity(property.getEntity());
+        property3.addTag("t3", "tv3");
+        property3.addKey("k3", "kv3");
+        insertPropertyCheck(property3);
+
+
+
```
Comment (oik77) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: Why 3 empty lines?

Comment (Sild) [inline src/test/java/com/axibase/tsd/api/method/property/PropertyQueryTest.java]: you're so greed

**Comment by Sild** [comment]
May I merge it?

**Comment by rmakulov** [comment]
Ok.

**Pull request merged**